src/Models/Nonhydrostatic3DModels/equations_base_model.jl

```diff
@@ -135,10 +135,10 @@ function rhs_base_model!(
         @. dρ += flux_correction_center(w, ρ)
     end
 
-    # discrete stiffness summation for spectral operations
-    Spaces.weighted_dss!(dρ)
-    Spaces.weighted_dss!(duh)
-    Spaces.weighted_dss!(dw)
+    # discrete stiffness summation for spectral operations, distributed
+    Spaces.weighted_dss!(dρ, comms_ctx_center)
```

Comment:
We are currently using a conservatively sized comms_ctx for both face and center DSS in ClimaCore